test/ecto/migrator_test.exs

```diff
@@ -169,7 +194,28 @@ defmodule Ecto.MigratorTest do
     """
   end
 
-  @tag :current
+  test "anonymous function migration" do
+    level = :info
+    num = System.unique_integer([:positive])
+    args = [TestRepo, num, AnonymousFunctionMigration, [log: level]]
+
+    for {name, direction} <- [migrate: :up, rollback: :down] do
```

Comment:
Because there is an execute in there without the second argument, I was expecting the rollback command to fail. I think adding the `reverse` instruction broke it. So can you please add a test that checks `execute/1` cannot be reverted?